### PR TITLE
Allows null options on relationship

### PR DESCRIPTION
**Pull request by tjmckenzie**
If field row is not required, provide None/Null option for BelongsTo and BelongsToMany

### CODE REVIEW

File: resources/views/formfields/relationship.blade.php [hunk 1]
```diff
@@ -29,8 +29,13 @@
 				<select class="form-control select2" name="{{ $options->column }}">
 					@php 
 						$model = app($options->model);
-	            		$query = $model::all();
-	            	@endphp
+						$query = $model::all();
+					@endphp
+
+					@if($row->required === 0)
+						<option value="">None</option>
```
Comment (cyrildewit) [inline resources/views/formfields/relationship.blade.php]: Would it be an idea to replace the static `None` with an automatic translation?

**Review by cyrildewit — CHANGES_REQUESTED** [review]

File: resources/views/formfields/relationship.blade.php (hunk 1 above)
Comment (emptynick) [reply resources/views/formfields/relationship.blade.php]: Theres already a translation. `generic.none` iirc

Comment (tjmckenzie) [reply resources/views/formfields/relationship.blade.php]: Good idea. I have just added it. I was glad to see "none" was already described in the generic translation files.

**Review by cyrildewit — APPROVED** [review]
Looks good to me!

**Comment by tjmckenzie** [comment]
Any ETA on a merge / release?

**Review by fletch3555 — APPROVED** [review]

**Pull request merged**